src/app/app-routing.module.ts

```diff
@@ -1,14 +1,37 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
 import { GameComponent } from './components/game.component';
-
+import { AccountComponent } from './components/account/account.component';
+import { TacticsTrainer } from './components/tactics-trainer/tactics-trainer.component';
+import { RepertoireBuilder } from './components/repertoire-builder/repertoire-builder.component';
+import { MyStudies } from './components/studies/my-studies/my-studies.component';
+import { PublicStudies } from './components/studies/public-studies/public-studies.component';
```

Comment:
The standard for component naming is SomethingComponent
only account is following this standard the rest of your components are just the names